src/components/pre-playback-play-overlay/_pre-playback-play-overlay.scss

```diff
@@ -29,6 +29,12 @@
       opacity: 0.7;
       transform: scale(1) translate(-50%, -50%);
     }
+
+    &:focus {
+      outline: 3px solid $tab-focus-color !important;
```

Comment:
why !important ? which style are you overriding ?

Reply:
the regular outline here - https://github.com/kaltura/playkit-js-ui/blob/master/src/styles/_shell.scss#L69